mlir/test/Integration/GPU/CUDA/command-line-arg.mlir

```diff
@@ -16,6 +16,7 @@ func.func @host_function(%arg0 : f32, %arg1 : memref<?xf32>) {
     return
 }
 
+// CHECK: ptxas executable:
```

Comment:
I'm puzzled that we have this test in-tree, in general we never test debug output, and I don't understand how is this test passing in release mode?

Reply:
I add a test for every code I write, so I probably added these test automatically :) 

This is a CUDA integration test. Perhaps we don't run CUDA integration tests in release mode?

Reply:
It's possible, can you remove it please?

Reply:
I removed from this PR. 
I have written other tests as well for debug runs. let me put a new PR that removes them all.